src/projects/yaq/src/index.html

```diff
@@ -0,0 +1,156 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Quiz App</title>
+    <link rel="stylesheet" href="public/styles/styles.css" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Inter:wght@400;600;700&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <div class="app">
+      <header>
+        <div class="greeting">
+          <div class="good-morning">
+            <img src="public/svg/sun.svg" alt="" />
+            <span> GOOD MORNING</span>
```

Comment:
will accept this extra spacing here for now

Reply:
I will fix that